src/diffusers/utils/import_utils.py

```diff
@@ -54,12 +54,17 @@
 _is_google_colab = "google.colab" in sys.modules or any(k.startswith("COLAB_") for k in os.environ)
 
 
-def _is_package_available(pkg_name: str):
+def _is_package_available(pkg_name: str, get_dist_name: bool = False) -> tuple[bool, str]:
```

Comment:
```suggestion
def _is_package_available(pkg_name: str, get_dist_name: bool = False) -> Tuple[bool, str]:
```
And `from typing import Tuple` if not already imported, we need to still support 3.8